Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -54,8 +54,7 @@ extension NSString {
             //
             // A reference to `NSString` is held by every cast `String` along with their views and
             // indices.
-            let cString = string.cStringUsingEncoding(NSUTF8StringEncoding)
-            let string = String(CString: cString, encoding: NSUTF8StringEncoding)!
+            let string = string.mutableCopy() as! String
```

Comment:
`NSString.copy` returns `self` because that is immutable. `NSString.mutableCopy` returns new instance of `NSString` with copied buffer. On this case, the `string` holds reference of the `CacheContainer` through associated object.
On baa8a3a, `ByteOffsetCache`(renamed to `CacheContainer`) held reference of `string` by [holding `String.UTF8Index`](https://github.com/norio-nomura/SourceKitten/blob/baa8a3a/Source/SourceKittenFramework/String%2BSourceKitten.swift#L34) and that caused circular reference.
On 6656224, it resolved by changing to [create new instance](https://github.com/norio-nomura/SourceKitten/blob/6656224/Source/SourceKittenFramework/String%2BSourceKitten.swift#L51-L52).
Using `mutableCopy()` replaces that.